src/content/docs/infrastructure/infrastructure-integrations/get-started/introduction-infrastructure-integrations.mdx

```diff
@@ -131,111 +133,21 @@ On-host integrations are infrastructure monitoring integrations that can be run
       </td>
 
       <td>
-        To create your own lightweight integration, use our [Flex integration tool](/docs/integrations/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration).
+        To create your own lightweight infrastructure integration, use our [Flex integration](/docs/integrations/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration).
       </td>
     </tr>
   </tbody>
 </table>
 
-## Install instructions [#installation-instructions]
-
-To enable cloud integrations or install on-host integrations, see:
-
-* **Cloud integrations**: [AWS procedures](/docs/infrastructure/amazon-integrations/getting-started/connect-aws-services-infrastructure), [Azure procedures](/docs/infrastructure/microsoft-azure-integrations/getting-started/activate-azure-integrations), [Google Cloud Platform procedures](/docs/integrations/google-cloud-platform-integrations/getting-started/connect-google-cloud-platform-services-infrastructure)
-* **Kubernetes**: [Kubernetes procedures](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration)
-* **On-host integrations**: See an [integration's documentation](/docs/integrations/host-integrations) for install procedures
-
 ## Features [#whats-next]
 
-After an infrastructure integration is activated, you can:
+After an infrastructure integration is enabled, you can:
 
-* [Filter and analyze](/docs/infrastructure/integrations/find-use-infrastructure-integration-data) the metrics and configuration data in [our Infrastructure UI](/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure).
+* [Filter and analyze](/docs/infrastructure/integrations/find-use-infrastructure-integration-data) your metrics and configuration data in [our infrastructure monitoring UI](/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure).
```

Comment:
It's pretty amazing be still have Infrastructure in capitals after so many sweeps...